### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bermudezmt**
Add a table describing the repository templates.

### CODE REVIEW

**Comment by bermudezmt** [comment]
A couple questions, @vernig or @kaiquelupo :

- Did we want to add the Twilio logo here or avoid doing that?
- Anything else we'd want to cover here? A brief overview on Twilio Flex and a link to Pulumi docs?

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
...
+</a>
+
+# Twilio Infrastructure as Code Examples
+
+This repository contains examples on how to deploy and manage Twilio Projects using Pulumi.
+
+## Checking Out a Single Example
+
+You can check out only the examples you want by clicking on the subfolder for the Flex plugin [repository template](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template) that you want, or clicking on the template names on the following table:
+
```
Comment (vernig) [inline README.md:12]: How about changing this one to: 

This project is organized as a collection of link to other repos that contain examples of how to use Pulumi to deploy Twilio projects. The best way to use this repo, is to open the repo using the ink in the following table, and click on ["Use this template"](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template)

**Comment by vernig** [comment]
@bermudezmt my two cents: 

> Did we want to add the Twilio logo here or avoid doing that?

My only concern with that is that we are all Twilions, and even if there is a disclaimer at the end of the README, it can be confused as official project. How about we use the "Powered by Twilio" logo instead: https://www.twilio.com/brand/elements/co-branding

> Anything else we'd want to cover here? A brief overview on Twilio Flex and a link to Pulumi docs?

I think this is really good as it is. I feel this repo should be linked from the README of both the twilio-pulumi-provider and plugin-twilio-infra. What do you think?

**Comment by vernig** [comment]
LGTM! Thanks @bermudezmt

**Pull request merged**